specs/derivation.md

```diff
@@ -473,28 +473,31 @@ updated, such that the batch-sender authentication is always accurate to the exa
 
 ### L1 Retrieval
 
-In the *L1 Retrieval* stage, we read the block we get from the outer stage (L1 traversal), and extract data from it.
-By default, the rollup operates on calldata retrieved from [batcher transactions][g-batcher-transaction] in the block,
-for each transaction:
+In the *L1 Retrieval* stage, we read the block we get from the outer stage (L1 traversal), and
+extract data from its [batcher transactions][g-batcher-transaction]. A batcher
+transaction is one with the following properties:
 
-- The receiver must be the configured batcher inbox address.
-- The sender must match the batcher address loaded from the system config matching the L1 block of the data.
+- The receiver is the configured batcher inbox address.
+- The sender is the batcher address loaded from the system config matching the L1 block of the data.
```

Comment:
This could also be more clear. There is no definition of sender. We could say something like the address that is recovered from the transaction signature (v, r and s). We want to be explicit that its the top level signature and not a metatx signature

Reply:
done. ptal

Reply:
Thank you!